src/classes/CON_CannotDelete_TEST.cls

```diff
@@ -50,7 +50,7 @@ public with sharing class CON_CannotDelete_TEST {
         Affiliation__c affl = new Affiliation__c(Contact__c = contacts[0].ID, Account__c = acc.ID);
         Program_Enrollment__c programEnroll = new Program_Enrollment__c(Contact__c = contacts[1].ID, Account__c = acc.ID);
         Course_Enrollment__c courseEnroll = new Course_Enrollment__c(Contact__c = contacts[2].ID, Account__c = acc.ID);
-        insert new SObject[] {affl, programEnroll, courseEnroll};
+        insert new SObject[] {affl, programEnroll, courseEnroll}; 
```

Comment:
Would it be possible to add additional tests here for not deleting a contact tied to a term grade or attendance event?